src/Silex/Provider/SecurityServiceProvider.php

```diff
@@ -114,7 +114,7 @@ public function register(Container $app)
         };
 
         $app['security.encoder.digest'] = function ($app) {
```

Comment:
Sorry. But wont have time for it this weekend and next week. 
Thanks for the feedback

Sent from my iPhone

> On 15 Feb 2016, at 10:17, Mathieu Rochette <notifications@github.com> wrote:
> 
> In src/Silex/Provider/SecurityServiceProvider.php:
> 
> > @@ -114,7 +114,7 @@ public function register(Container $app)
> >          };
> >  
> >          $app['security.encoder.digest'] = function ($app) {
> bonus, $app['security.encoder.bcrypt.cost'] could be used to configure/override the default encoder cost
> 
> —
> Reply to this email directly or view it on GitHub.
>